docs/guides/theme_architecture/blocks/schema/variables/variable_types/radio.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: default
+title: Radio
+parent: Variable types
+---
+
+# Radio
+
+Renders a radio button group and returns the value of the selected option.
+Takes an `options` parameter which defines the options in the radio button group, this is an array of label, value pairs.
+
+##### syntax
+{% raw %}
+```
+---
+my_variable:
+    type: radio
+    default: opt_1
+    options:
+        - label: Option 1
+          value: opt_1
+        - label: Option 2
+          value: opt_2
+---
+
+{{my_variable}}
```

Comment:
would use `{{my_variable.value}}` in the liquid?

Reply:
The radio and select just return the value, with no option to return the label